cli/package.json

```diff
@@ -2,17 +2,25 @@
   "name": "stoat",
   "version": "0.0.8",
   "description": "Stoat CLI",
-  "main": "src/index.ts",
+  "main": "dist/index.js",
```

Comment:
The `src/index.ts` file will be included in the final package if it is the `main`. However, a `ts` file is not usable in the package. So I think technically the correct main is `dist/index.js`.